cli/zone.go

```diff
@@ -93,7 +92,7 @@ func runGetZone(cmd *cobra.Command, args []string) {
 		log.Errorf("Object %d: no zone config found", id)
 		return
 	}
-	fmt.Fprintln(osStdout, rows[0][1])
+	fmt.Println(rows[0][1])
```

Comment:
Ouch. Let's file a follow-up issue to move `sql_util` into its own package so that `osStdout` is scoped to just the package that needs it. The current code is pretty error prone in light of this knowledge.